ibridgesgui/sync.py

```diff
@@ -81,7 +88,7 @@ def _init_irods_tree(self):
 
     def irods_root(self, irods_path):
         """Retrieve lowest visible level in the iRODS tree for the user."""
-        lowest = IrodsPath(irods_path.session, irods_path.absolute_path())
+        lowest = IrodsPath(irods_path.session, irods_path.absolute())
         while lowest.parent.exists():
             lowest = IrodsPath(irods_path.session, lowest.parent)
```

Comment:
Ah, the iRODS home can be somewhere in the tree, but I really want to show the the tree in the panel from its root `/`, in case the user wants to transfer data that lies outside of the irods_home. And since I do not know where in the tree the irods_home is, I need to recurse down.

Maybe it can be done more elegantly.